src/ui/.prettierrc

```
@@ -0,0 +1,13 @@
+{
+  "arrowParens": "always",
+  "bracketSameLine": false,
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "jsxSingleQuote": false,
+  "printWidth": 100,
```

```suggestion
  "printWidth": 88,
```

Let's set this to the same as the python code base is